packages/libs/kode-ui/src/components/Form/Select/Select.css.ts

```diff
@@ -13,24 +14,28 @@ export const selectButtonValue = style({
   flex: 1,
 });
 
-export const selectButtonClass = style({
-  display: 'flex',
-  backgroundColor: token('color.background.input.default'),
-  justifyContent: 'space-between',
-  color: token('color.text.base.default'),
-  flex: 1,
-  overflow: 'hidden',
-  position: 'relative',
-  alignItems: 'center',
-  paddingBlock: 0,
-  paddingInlineStart: token('spacing.md'),
-  paddingInlineEnd: 0,
-  selectors: {
-    '&[data-hovered]': {
-      cursor: 'pointer',
+export const selectButtonClass = style([
+  atoms({
+    paddingBlock: 'no',
+    flex: 1,
+    overflow: 'hidden',
+    position: 'relative',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  }),
+  {
+    display: 'flex',
+    backgroundColor: token('color.background.input.default'),
```

Comment:
if the background color would be defined in `atoms` the `token` wouldn't be necessary right?

Reply:
created a story to cleanup the CSS
https://app.asana.com/0/1206752931509995/1208343913799309/f